src/cfclient/ui/dialogs/inputconfigdialogue.py

```diff
@@ -178,7 +183,8 @@ def _reset_mapping(self):
             "pitchNeg": self.pitchNeg,
             "rollPos": self.rollPos,
             "rollNeg": self.rollNeg,
-            "killswitch": self.killswitch,
+            "estop": self.killswitch,
```

Comment:
I was wondering... do we want to force people to choose a kill button? Like, they can not save the config file if there is no pitch yaw roll and thrust assigned... but we can also force people to choose a kill button. This needs to be for both the crazyflie or bolt based systems I suppose.

Reply:
Might be a good idea but could there be joysticks without buttons? Then it would block those, which might be good...